File: Python/SQLAlchemy_ORM/dal/entities/employee.py
from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Text
from sqlalchemy.orm import relationship
from dal.data.base import Base

class Employee(Base):
    __tablename__ = 'Employees'

    EmployeeID = Column(Integer, primary_key=True)
    LastName = Column(String(20))
    FirstName = Column(String(10))
    Title = Column(String(30))
    TitleOfCourtesy = Column(String(25))
    BirthDate = Column(DateTime)
    HireDate = Column(DateTime)
    Address = Column(String(60))
    City = Column(String(15))
    Region = Column(String(15))
    PostalCode = Column(String(10))
    Country = Column(String(15))
    HomePhone = Column(String(24))
    Extension = Column(String(4))
    Notes = Column(Text)
    Photo = Column(String)
    PhotoPath = Column(String(255))

    reports_to = Column(Integer, ForeignKey('Employees.EmployeeID'))
    #reports_to_navigation = relationship('Employee', remote_side=[Employee.EmployeeID], backref='inverse_reports_to_navigation')
    territories = relationship('Territory', secondary='EmployeeTerritories', back_populates='employees')
    orders = relationship('Order', back_populates='employee')

File: Go/GORM_ORM/dal/entities/employee.go
package entities

import "time"

type Employee struct {
	EmployeeID      int        `gorm:"primaryKey;autoIncrement;column:EmployeeID"`
	LastName        string     `gorm:"size:20;index:LastName;column:LastName"`
	FirstName       string     `gorm:"size:10;column:FirstName"`
	Title           *string    `gorm:"size:30;column:Title"`
	TitleOfCourtesy *string    `gorm:"size:25;column:TitleOfCourtesy"`
	BirthDate       *time.Time `gorm:"column:BirthDate"`
	HireDate        *time.Time `gorm:"column:HireDate"`
	Address         *string    `gorm:"size:60;column:Address"`
	City            *string    `gorm:"size:15;column:City"`
	Region          *string    `gorm:"size:15;column:Region"`
	PostalCode      *string    `gorm:"size:10;index:PostalCode;column:PostalCode"`
	Country         *string    `gorm:"size:15;column:Country"`
	HomePhone       *string    `gorm:"size:24;column:HomePhone"`
	Extension       *string    `gorm:"size:4;column:Extension"`
	Photo           *[]byte    `gorm:"type:image;column:Photo"`
	Notes           *string    `gorm:"type:ntext;column:Notes"`
	ReportsTo       *int       `gorm:"column:ReportsTo"`
	PhotoPath       *string    `gorm:"size:255;column:PhotoPath"`
	Manager         *Employee  `gorm:"foreignKey:ReportsTo"`
	Subordinates    []Employee `gorm:"foreignKey:ReportsTo"`
	Territories     []EmployeeTerritory
	Orders          []Order `gorm:"foreignKey:EmployeeID"`
}
